hamocc/mo_param_bgc.F90

```diff
@@ -656,8 +656,8 @@ subroutine ini_param_biol()
       ropal  = 10.5        ! opal to organic phosphorous production ratio
       calmax = 0.20
     else if (use_WLIN) then
-      rcalc  = 33.         ! calcium carbonate to organic phosphorous production ratio
-      ropal  = 45.         ! opal to organic phosphorous production ratio
+      rcalc  =  8.         ! calcium carbonate to organic phosphorous production ratio
+      ropal  = 70.         ! opal to organic phosphorous production ratio
```

Comment:
Hi @JorgSchwinger , just for information this is a pretty high value compared to literature (while not totally out of range). Typical range likely between ca. 4-51, mean ca. 16, maximum ca. 99 according to Nelson et al. 1995.